src/GovUk.Education.ExploreEducationStatistics.Public.Data.Model/DataSetVersion.cs

```diff
@@ -101,7 +109,9 @@ public void Configure(EntityTypeBuilder<DataSetVersion> builder)
                     );
             });
 
-            builder.Property(dsv => dsv.Status).HasConversion<string>();
+            builder.HasIndex(dsv => new { dsv.DataSetId, dsv.VersionMajor, dsv.VersionMinor, dsv.VersionPatch })
+                .HasDatabaseName("IX_DataSetVersions_DataSetId_VersionNumber")
+                .IsUnique();
```

Comment:
In the changes in `InitialCreate` it looks like this change is removing the index on `DataSetId` in favour of only having the multicolumn index on all the columns combined.

I was expecting this change only to create an additional index, rather than remove the exising `DataSetId` one.

Reading the docs it looks like this is ok as they say the search effectiveness is the same regardless of which index column(s) the query conditions use, but just thought I'd mention it.

Reply:
Yeh I noticed this, but EF was originally setting up the `DataSetId` index for us. I don't know if EF decided it didn't need to set this up as it was redundant, or because I was manually setting this index that affected `DataSetId`.

Either way, as you've said though, I don't think this will make any substantial difference so think it's fine to leave it is as-is.

Reply:
Cool, hopefully it realised the new one encompasses the old one and that it's not needed. 🙂